### ISSUE TITLE
`Encrypt ballot` produces an error message in the web ui

### ISSUE THREAD

**Issue by RaisinTen** [stduhpf/caritat#4]
After filling https://stduhpf.github.io/caritat/#https://github.com/nodejs/TSC/pull/1158 with all my details, when I click on `Encrypt ballot`, an error message shows up at the bottom of the page:

> An error occured: The request is not allowed by the user agent or the platform in the current context, possibly because the user denied permission.

I noticed this only after I committed the faulty vote I reported in https://github.com/stduhpf/caritat/issues/3, not sure if it happens otherwise.

**Comment by aduh95** [comment]
FWIW we are using the git commit author, it's repeated in the JSON file for convenience but it has no implication on your vote. The web UI can't guess what is the name or the email address of the voter, so it's simpler easier to not include it.

Closing as won't fix, thanks for the report.

**Comment by aduh95** [comment]
Sorry commented on the wrong thread, reopening.

**Comment by aduh95** [comment]
Can you share what browser you are using? Also, it looks like you managed to vote anyway, is it because the error is not blocking?

**Comment by RaisinTen** [comment]
I'm using `Safari Version 15.2 (17612.3.6.1.6)`. The encrypted ballot does get generated, which is what I used in the commit but I wasn't sure if that contains valid data, so I just wanted to reconfirm.

**Comment by aduh95** [comment]
The error was showing up because Safari doesn't want the web app to send data to the clipboard right away. This can be safely ignored, it has nothing to do with the ballot data :) I've pushed a fix to not display the error message before the user clicks the `Copy to clipboard` button.
